docs/design2.md

```diff
@@ -0,0 +1,102 @@
+# Key Transparency 2.0
+
+## Objective
+
+Key Transparency currently sufferes from a design-space tradeoff in two dimentions between
+speed and verification bandwidth.  Each snapshot of the key-value merkle tree creates an additional
+point in time that clients must audit when inspecting their account history. 
+Using Key Transparency in this mode requires the presence of powerful auditors who can verify
+that no information has been lost between all snapshots.
+
+| Update Speed | Trust, then Verifiy | Verify First |
+|--------------|---------------------|--------------|
+| hr           | Tractable Bandwidth | Unusable     |
+| min          |                     | Intractable  |
+| s            |                     | Intractable  |
+
+
+Key Transparency 2.0 is a new set of algorithms that resolves this fundamental tradeoff by sharing 
```

Comment:
Is it resolving the trade off, or removing the tradeoff?

Reply:
removes it